task5/src/main/java/com/clouway/adapter/persistence/PersistentSessionRepository.java

```diff
@@ -0,0 +1,82 @@
+package com.clouway.adapter.persistence;
+
+import com.clouway.core.ConnectionProvider;
+import com.clouway.core.Session;
+import com.clouway.core.SessionRepository;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * Created by Kristiyan Petkov  <kristiqn.l.petkov@gmail.com> on 02.06.16.
+ */
+public class PersistentSessionRepository implements SessionRepository {
+  private ConnectionProvider connectionProvider;
+
+  public PersistentSessionRepository(ConnectionProvider connectionProvider) {
+    this.connectionProvider = connectionProvider;
+  }
+
+  public void createSession(Session session) {
```

Comment:
Session that you store in DB there should be expirationTime that you can set in here.